<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockAvgCalculator</class>
 <widget class="QWidget" name="StockAvgCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>주식 평균 단가 계산기</string>
  </property>
  
  <widget class="QLabel" name="label_current_price">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>현재 평균 단가:</string>
   </property>
  </widget>
  
  <widget class="QLineEdit" name="input_current_price">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  
  <widget class="QLabel" name="label_new_price">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>추가 매수 가격:</string>
   </property>
  </widget>
  
  <widget class="QLineEdit" name="input_new_price">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  
  <widget class="QLabel" name="label_quantity">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>추가 매수 수량:</string>
   </property>
  </widget>
  
  <widget class="QLineEdit" name="input_quantity">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  
  <widget class="QPushButton" name="button_calculate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>계산하기</string>
   </property>
  </widget>
  
  <widget class="QLabel" name="label_result">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>360</width>
     <height>40</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="text">
    <string>결과: -</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
